.prettierrc

```diff
@@ -7,5 +7,13 @@
   "trailingComma": "none",
   "jsxBracketSameLine": false,
   "arrowParens": "avoid",
-  "jsxSingleQuote": false
+  "jsxSingleQuote": false,
+  "overrides": [
+    {
+      "files": ["*.ts", "**/*.ts"],
+      "options": {
+        "semi": true
```

Comment:
I'm guessing this is what adds the semicolons?

Reply:
Yes, and in eslint, removing now